third_party/binutils/test_decenc/test_decenc_x86.asm

```diff
@@ -73691,8 +73691,6 @@ GLOBAL_LABEL(FUNCNAME:)
         RAW(66) RAW(2e) RAW(0f) RAW(1f) RAW(84) RAW(00) RAW(00)
         RAW(00) RAW(00) RAW(00)
         RAW(0f) RAW(19) RAW(ff)
-        RAW(0f) RAW(1a) RAW(ff)
```

Comment:
Why are these lines being removed?  If these are bnd* instructions don't we want to test them?  Ditto below.

Reply:
This are not bnd* instructions, they are invalid because of the invalid registers. The original tests included invalid encodings, we are only testing valid ones.

Reply:
Please add a note on this to the commit message, as it is not clear why these test were changed.